paasta_tools/cli/utils.py

```diff
@@ -582,7 +585,8 @@ def execute_chronos_rerun_on_remote_master(service, instancename, cluster, syste
 
 
 def run_on_master(cluster, system_paasta_config, cmd_parts,
-                  timeout=None, err_code=-1, graceful_exit=False, stdin=None):
+                  timeout=None, err_code=-1, graceful_exit=False, stdin=None,
+                  random_master=False):
```

Comment:
don't bother with the random_master arg if we don't need it (I'm pretty sure we don't)